### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by joshmh**
Added CONTRIBUTING.md as per https://github.com/liberland/constitution/issues/17

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+If you want to fix a series of typos, you can submit a pull request that incorporates all of your typo changes in the document.
+
+## Adding a line
+
+If you want to add a new, numbered line to the constitution, add it as the last item of the relevant section. This way you won't have to re-number all the other items, which would make it very hard to review the change. Don't worry if the change would make more sense next to a higher-up item. That will be fixed after all the changes are discussed.
+
+## Removing a line
+
+If you'd like to remove a line, delete it but don't update the numbering of any of the other lines. This will be fixed in the end. For now, we want to keep your change restricted to the line you're changing. Make sure you fix any references to the item you deleted.
```
Comment (hazzik) [inline CONTRIBUTING.md]: I would like to say something like "$II.100 Is obsolete now" instead of just removing the line

Comment (joshmh) [inline CONTRIBUTING.md]: The diff will show the deletion. You can write that in the commit comment.

**Comment by burningtree** [comment]
I do not think it is necessary to guide all to use the command line. These repositories will contain only text files, so everything can be easily edited using GitHub web interface - this is not a software project.

We need help from the public, it is pointless to send lawyers to the command line.

**Comment by joshmh** [comment]
Oh cool, I didn't realize that creates a pull request. I'll put that in instead.

**Comment by burningtree** [comment]
Two files are unnecessary, please consolidate it into one.

File: CONTRIBUTING.md (hunk 1 above)
Comment (hazzik) [inline CONTRIBUTING.md]: The diff - yes, but the numering itself will lead to confusion. So, people just have to see that this point is no longer valid.

Comment (joshmh) [inline CONTRIBUTING.md]: Ok, that makes sense, I'll add it.

**Pull request merged**